en/components/icon-service.md

```diff
@@ -0,0 +1,233 @@
+| input_expand   | material      | expand_more          | Combo, Simple Combo, Select                             | Material                    |
+| input_collapse | material      | expand_less          | Combo, Simple Combo, Select                             | Material                    |
+| input_expand   | material      | arrow_drop_down      | Combo, Simple Combo, Select                             | Indigo, Fluent, Bootstrap   |
+| input_collapse | material      | arrow_drop_up        | Combo, Simple Combo, Select                             | Indigo, Fluent, Bootstrap   |
+| last_page      | material      | last_page            | Grid, Paginator                                         | All                         |
+| more_vert      | material      | more_vert            | Action Strip, Grid                                      | All                         |
+| next           | material      | navigate_next        | Grid, Tabs, Filtering Row                               | All                         |
+| prev           | material      | navigate_before      | Grid, Tabs, Filtering Row                               | All                         |
+| remove         | material      | cancel               | Grid, Chip                                              | All                         |
+| refresh        | material      | refresh              | Grid, Filtering Row                                     | All                         |
+| selected       | material      | done                 | Grid, Chip, Filtering Row                               | All                         |
+| table_rows     | material      | table_rows           | Grid                                                    | All                         |
+| today          | material      | today                | Date Picker                                             | All                         |
+| view_column    | material      | view_column          | Grid                                                    | All                         |
+| visibility     | material      | visibility           | Grid, Excel Style Filtering                             | All                         |
+| visibility_off | material      | visibility_off       | Grid, Excel Style Filtering                             | All                         |
+
+To take advantage of changing the internal icons by reference, as opposed to creating custom templates, you can do the following to replace the expand/collapse icons in the combo and select components:
+
+```ts
+setIconRef('input_expand', 'default', {
+    name: 'arrow_downward',
+    family: 'material',
+});
+
+setIconRef('input_collapse', 'default', {
+    name: 'arrow_upward',
+    family: 'material',
+});
+```
+
+This will set the expand icon to the `chevron_right` ligature from the `material` font family.
```

Comment:
```suggestion
This will set the expand and collapse icons to the `arrow_downward` and `arrow_upward` ligatures, respectively, from the `material` font family for all combo and select components.
```